GitUI/UserControls/CommitDiff.cs

```diff
@@ -76,14 +77,30 @@ private async void DiffText_ExtraDiffArgumentsChanged(object sender, EventArgs e
             }
         }
 
+        // Partly the same as RevisionDiffControl.cs ShowSelectedFileDiffAsync()
```

Comment:
The code change is based on RevisionDiffControl.cs ShowSelectedFileDiffAsync()
The diff handling should be updated, this is not adding much extra maintenance
(the primary alternative should be to keep the exception)